src/Components/Artwork/GridItem.tsx

```diff
@@ -71,6 +74,8 @@ class ArtworkGridItemContainer extends React.Component<Props, State> {
     if (isFunction(window.matchMedia)) {
       this.canHover = !window.matchMedia("(hover: none)").matches
     }
+    IMAGE_LAZY_LOADING =
+      sd.IMAGE_LAZY_LOADING || process.env.IMAGE_LAZY_LOADING === "true"
```

Comment:
This should also be added to the storybooks [webpack config](https://github.com/artsy/reaction/blob/master/.storybook/webpack.config.js#L20)